dipy/viz/streamline.py

```diff
@@ -1,12 +1,18 @@
+import warnings
 from dipy.utils.optpkg import optional_package
 
 plt, have_plt, _ = optional_package("matplotlib.pyplot")
 fury, has_fury, _ = optional_package('fury', min_version="0.10.0")
 if has_fury:
     from dipy.viz import actor, window
 
+sagital_deprecation_warning_msg = (
+    "The view argument value `sagital` is deprecated and its support will be"
```

Comment:
can you add `# codespell:ignore` even if it is not taken into account yet

Reply:
Added in b5be3fc. I think it goes after the closing bracket as it is considered the same line. At least that is the case for `noqa`.

Reply:
For noqa, you can have both. here in this case, not sure.

Reply:
OK, but for now, we simply do not know (unless we take the hassle of locally installing `master`, etc.); we will figure out and fix that tiny bit when the release comes out and see how it behaves.